### PR TITLE
[formrecognizer] working on tests

### LINKED ISSUE #10192: [FormRecognizer] Add sync/async tests and set up testing pipelines
- [x] Set up form recognizer CI pipeline
- [x] Set up preparer test framework for form recognizer
- [x] Write sync tests
- [x] Write async tests
- [x] Record tests and ensure playback works
- [ ] Set up live test pipeline and schedule nightly

### PR DESCRIPTION
**Pull request by kristapratico**
Resolves #10192 

Live test pass in playground: https://dev.azure.com/azure-sdk/playground/_build/results?buildId=357796&view=results

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_type.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# ------------------------------------
+
+import pytest
+from azure.ai.formrecognizer._helpers import get_content_type
+from testcase import FormRecognizerTest
+
+
+class TestContentType(FormRecognizerTest):
+
+    def test_pdf(self):
```
Comment (iscai-msft) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_type.py:14]: Can you add a test that ensures that passes a content type (wusing enum), and make sure it overwrites the suggest content type?

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_mgmt.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    def test_account_properties(self, resource_group, location, form_recognizer_account, form_recognizer_account_key):
+        client = FormTrainingClient(form_recognizer_account, AzureKeyCredential(form_recognizer_account_key))
+        properties = client.get_account_properties()
+
+        self.assertIsNotNone(properties.custom_model_limit)
+        self.assertIsNotNone(properties.custom_model_count)
+
+    @GlobalFormAndStorageAccountPreparer()
+    @GlobalTrainingAccountPreparer()
+    def test_mgmt_model(self, client, container_sas_url):
```
Comment (iscai-msft) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/test_mgmt.py:31]: I think you should split this up, so one for labeled, one for unlabeled

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/testcase.py [hunk 3]
```diff
@@ -5,51 +5,289 @@
...
+            self.assertIsNotNone(item.name.confidence)
+            self.assertIsNotNone(item.name.value_data.text)
+            self.assertBoundingBoxHasPoints(item.quantity.value_data.bounding_box)
+            self.assertIsNotNone(item.quantity.confidence)
+            self.assertIsNotNone(item.quantity.value_data.text)
+            self.assertBoundingBoxHasPoints(item.total_price.value_data.bounding_box)
+            self.assertIsNotNone(item.total_price.confidence)
+            self.assertIsNotNone(item.total_price.value_data.text)
+
+            if include_text_content:
```
Comment (iscai-msft) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/testcase.py:235]: you can also assert that if not include_text_content, these properties are None

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_type.py (hunk 1 above)
Comment (kristapratico) [reply sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_type.py:14]: done

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_mgmt.py (hunk 2 above)
Comment (kristapratico) [reply sdk/formrecognizer/azure-ai-formrecognizer/tests/test_mgmt.py]: you noticed my laziest. :) Done.

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/testcase.py (hunk 3 above)
Comment (kristapratico) [reply sdk/formrecognizer/azure-ai-formrecognizer/tests/testcase.py:235]: done

**Pull request merged**